Source/Shared/EntityAttribute.cs

```diff
@@ -0,0 +1,8 @@
+using JetBrains.Annotations;
+
+namespace CodeImp.Bloodmasters;
+
+[MeansImplicitUse]
+public abstract class EntityAttribute : Attribute
```

Comment:
Maybe it's worth leaving some comment that classes annotated with such attribute are instantiated dynamically in runtime?

Reply:
Sure, why not.